packages/eslint-plugin/tests/rules/prefer-includes.test.ts

```diff
@@ -13,165 +11,140 @@ const ruleTester = new RuleTester({
   },
 });
 
-type MessageIds = util.InferMessageIdsTypeFromRule<typeof rule>;
-
-type InvalidTestCase = TSESLint.InvalidTestCase<MessageIds, never>;
-type ValidTestCase = TSESLint.ValidTestCase<never> | string;
-function addOptional(cases: ValidTestCase[]): ValidTestCase[];
-function addOptional(cases: InvalidTestCase[]): InvalidTestCase[];
-function addOptional(
-  cases: (ValidTestCase | InvalidTestCase)[],
-): (ValidTestCase | InvalidTestCase)[] {
-  return cases.reduce<(ValidTestCase | InvalidTestCase)[]>((acc, c) => {
-    acc.push(c);
-    if (typeof c === 'string') {
-      acc.push(c.replace('.', '?.'));
-    } else {
-      acc.push({
-        ...c,
-        code: c.code.replace('.', '?.'),
-        output: 'output' in c ? c.output?.replace('.', '?.') : null,
-      });
-    }
-
-    return acc;
-  }, []);
-}
```

Comment:
This `addOptional` wrapper was hiding the test cases from being checked by Prettier... although I could keep it in and remove the `output` field, it does feel weird to me to auto-generate them this way when the only difference between the two is checking `callNode.parent.parent` vs. `callNode.parent`?

Reply:
this was just my lazy way of covering all the cases 😅